bin/GameIndex.yaml

```diff
@@ -39441,17 +39519,20 @@ SLUS-29170:
 SLUS-29171:
   name: "Final Fantasy XII [Demo]"
   region: "NTSC-U"
+SLUS-29172:
+  name: "Battlefield 2: Modern Combat [Demo]"
+  region: "NTSC-U"
 SLUS-29173:
   name: "Sims 2, The [Demo]"
   region: "NTSC-U"
 SLUS-29174:
-  name: "Namco Transmission Demo Disc Vol.3.1"
+  name: "Namco Transmission Demo Disc Vol. 3.1"
   region: "NTSC-U"
 SLUS-29175:
-  name: "Namco Transmission Demo Disc Vol.3.2"
+  name: "Namco Transmission Demo Disc Vol. 3.2"
   region: "NTSC-U"
 SLUS-29178:
-  name: "TOCA Race Driver 3 [Demo]"
+  name: "TOCA Race Driv3r [Demo]"
```

Comment:
As before Driv3r only exists in the Driver universe, not every game with "Driver 3" in the title.